src/postgraphile/withPostGraphileContext.ts

```diff
@@ -104,7 +105,7 @@ const withDefaultPostGraphileContext: WithPostGraphileContextFn = async (
 
   const { role: pgRole, localSettings, jwtClaims } = getSettingsForPgClientTransaction({
     jwtToken,
-    jwtSecret,
+    jwtSecretOrPublicKey: jwtPublicKey || jwtSecret,
```

Comment:
Lets just pass through values directly right up until they're used.

```suggestion
    jwtSecret,
    jwtPublicKey,
```